tools/v8-prof/linux-tick-processor

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+TEMP_SCRIPT_FILE="/tmp/node-tick-processor-input-script"
+tools_path=`cd $(dirname "$0");pwd`
+v8_tools="$tools_path/../../deps/v8/tools"
+log_file="v8.log"
+for arg in "$@"
+do
+  if ! expr "X${arg}" : "^X-" > /dev/null; then
```

Comment:
This seems to give warnings on some platforms, for instance: https://jenkins-iojs.nodesource.com/job/iojs+pr+linux/nodes=centos5-32/101/tapTestReport/test.tap-660/